docs/advanced-configuration.md

```diff
@@ -399,15 +399,36 @@ does all the necessary metadata enrichment.
 
 OpenTelemetry Collector also has
 [native functionality for logs collection](https://github.com/open-telemetry/opentelemetry-log-collection).
-This chart soon will be migrated from fluentd to the OpenTelemetry logs collection.
 
-You already have an option to use OpenTelemetry logs collection instead of fluentd.
-The following configuration can be used to achieve that:
+**IMPORTANT:** This chart will be migrated from Fluentd to the OpenTelemetry logs collection by default starting from
+version 0.86.0. Please ensure that "logsEngine" parameter is explicitly set to avoid unexpected changes during upgrade.
+
+The following configuration can be used to switch between fluentd and OpenTelemetry logs collection:
 
 ```yaml
-logsEngine: otel
+logsEngine: <fluentd|otel>
 ```
 
+### Difference between fluentd and OpenTelemetry logs collection
+
+#### Emitted logs
+
+There is almost no difference in the logs emitted by default between the two engines. The only difference is that
+fluentd logs have an additional attribute called `fluent.tag` which has a value similar to `source` HEC field.
+
```

Comment:
There is almost no difference in the logs emitted by default by the two engines. The only difference is that
Fluentd logs have an additional attribute called ``fluent.tag``, which has a value similar to the ``source`` HEC field.

Reply:
Thanks